string id #Unique name for the action
string description # Optional description of the action
string group_name # Name of the planning group the action is for
waypoint_manager_msgs/Motion[] motions